frappe/desk/treeview.py

```diff
@@ -40,22 +43,15 @@ def get_children(doctype, parent='', **filters):
 
 def _get_children(doctype, parent='', ignore_permissions=False):
 	parent_field = 'parent_' + doctype.lower().replace(' ', '_')
-	filters = [['ifnull(`{0}`,"")'.format(parent_field), '=', parent],
-		['docstatus', '<' ,'2']]
 
-	meta = frappe.get_meta(doctype)
+	DocType = frappe.qb.DocType(doctype)
 
-	return frappe.get_list(
-		doctype,
-		fields=[
-			'name as value',
-			'{0} as title'.format(meta.get('title_field') or 'name'),
-			'is_group as expandable'
-		],
-		filters=filters,
-		order_by='name',
-		ignore_permissions=ignore_permissions
-	)
+	meta = frappe.get_meta(doctype)
+	title_field = meta.get('title_field') or 'name'
+	title = Field(title_field)
+	return (frappe.qb.from_(DocType)
+		.select(DocType.name.as_('value'), title.as_('title'), DocType.is_group.as_('expandable'))
+		.where((Coalesce(parent_field, '') == parent) & (DocType.docstatus < 2))).run()
```

Comment:
This change renders the `ignore_permissions` kwarg useless. I think we need to revert this back to `frappe.get_list` for the permission checks.

Reply:
👍

Reply:
Now reverted back to get_list